pkg/agent/agent_windows.go

```diff
@@ -142,6 +142,10 @@ func (i *Initializer) prepareOVSBridge() error {
 	if _, err = i.ovsBridgeClient.GetOFPort(brName); err == nil {
 		klog.Infof("OVS bridge local port %s already exists, skip the configuration", brName)
 	} else {
+		// Rename the vnic created by Windows host with the OVS bridge name, then it can be used by OVS directly.
+		if err := util.RenameVMNetworkAdapter(hnsNetwork.Name, uplinkMACStr, brName); err != nil {
+			return err
+		}
```

Comment:
I assume we never need to undo this change? deleting the HNSNetwork will still take care of all the clean-up?

Reply:
You are correct, the vnic created by Windows host is removed even if we have renamed it when the HNSNetwork is removed.